lib/models/category.js

```diff
@@ -30,7 +30,8 @@ module.exports = function(ctx) {
 
   Category.virtual('path').get(function() {
     var catDir = ctx.config.category_dir;
-    if (catDir[catDir.length - 1] !== '/') catDir += '/';
+    if (catDir === '/') catDir = '';
+    if (catDir.length > 0 && catDir[catDir.length - 1] !== '/') catDir += '/';
```

Comment:
what you mean? I know the evaluation is the same, I used explicit comparison for better reading.
Have I remove `> 0`? If Y, no prob.